app/routes/components.alert.tsx

```diff
@@ -121,24 +120,14 @@ export default function Page() {
 						<TableRow>
 							<TableCell className="align-top font-medium">priority</TableCell>
 							<TableCell className="space-y-2 align-top text-xs">
-								<p>
-									<InlineCode>&quot;danger&quot;</InlineCode>
```

Comment:
i think it's probs important to have it appear like `"danger"` since it's the string `"danger"` in code